docs/storage_snippets.py

```diff
@@ -0,0 +1,257 @@
+    # None
+    # [END get_blob]
+
+    to_delete.append(bucket)
+
+
+@snippet
+def delete_blob(client, to_delete):
+    # [START delete_blob]
+    from google.cloud.exceptions import NotFound
+    client = storage.Client()
+    bucket = client.get_bucket('my-bucket')
+    print(bucket.list_blobs())
+    # [<Blob: my-bucket, my-file.txt>]
+    bucket.delete_blob('my-file.txt')
+    try:
+        bucket.delete_blob('doesnt-exist')
+    except NotFound:
+        pass
+    # [END delete_blob]
+
+    blob = None
+    # [START delete_blobs]
+    bucket.delete_blobs([blob], on_error=lambda blob: None)
+    # [END delete_blobs]
+
+    to_delete.append(bucket)
+
+
+@snippet
+def configure_website(client, to_delete):
+    bucket_name = 'test-bucket'
+    # [START configure_website]
+    client = storage.Client()
+    bucket = client.get_bucket(bucket_name)
+    bucket.configure_website('index.html', '404.html')
+    # [END configure_website]
+
+    # [START make_public]
+    bucket.make_public(recursive=True, future=True)
+    # [END make_public]
+
+    to_delete.append(bucket)
+
+
+@snippet
+def get_bucket(client, to_delete):
+    import google
+    # [START get_bucket]
+    try:
+        bucket = client.get_bucket('my-bucket')
+    except google.cloud.exceptions.NotFound:
+        print('Sorry, that bucket does not exist!')
+    # [END get_bucket]
+    to_delete.append(bucket)
+
+
+@snippet
+def lookup_bucket(client, to_delete):
+    # [START lookup_bucket]
+    bucket = client.lookup_bucket('doesnt-exist')
+    print(bucket)
+    # None
+    bucket = client.lookup_bucket('my-bucket')
+    print(bucket)
+    # <Bucket: my-bucket>
+    # [END lookup_bucket]
+
+    to_delete.append(bucket)
+
+
+@snippet
+def create_bucket(client, to_delete):
+    # [START create_bucket]
+    bucket = client.create_bucket('my-bucket')
+    print(bucket)
+    # <Bucket: my-bucket>
+    # [END create_bucket]
+
+    to_delete.append(bucket)
+
+
+@snippet
+def list_buckets(client, to_delete):
+    # [START list_buckets]
+    for bucket in client.list_buckets():
+        print(bucket)
+    # [END list_buckets]
+
+    for bucket in client.list_buckets():
+        to_delete.append(bucket)
+
+
+def _line_no(func):
+    code = getattr(func, '__code__', None) or getattr(func, 'func_code')
+    return code.co_firstlineno
+
+
+def _find_examples():
+    funcs = [obj for obj in globals().values()
+             if getattr(obj, '_snippet', False)]
+    for func in sorted(funcs, key=_line_no):
+        yield func
+
+
+def _name_and_doc(func):
+    return func.__name__, func.__doc__
+
+
+def main():
+    client = storage.Client()
+    for example in _find_examples():
+        to_delete = []
+        print('%-25s: %s' % _name_and_doc(example))
+        try:
+            example(client, to_delete)
+        except AssertionError as e:
+            print('   FAIL: %s' % (e,))
+        except Exception as e:  # pylint: disable=broad-except
+            print('  ERROR: %r' % (e,))
```

Comment:
Replace the `e` variable with a longer name (as in https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2556)